GovUkDesignSystemComponents/SummaryListRowViewModel.cs

```diff
@@ -0,0 +1,117 @@
+﻿using GovUkDesignSystem.GovUkDesignSystemComponents.SubComponents;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GovUkDesignSystem.GovUkDesignSystemComponents
+{
+    public class SummaryListRowViewModel
+    {
+        /// <summary>
+        ///     Classes to add to the row div.
+        /// </summary>
+        public string Classes { get; set; }
+
+        /// <summary>
+        ///     Add row key. 
+        /// </summary>
+        public SummaryListRowKey Key { get; set; }
+
+        /// <summary>
+        ///     Add row value. 
+        /// </summary>
+        public SummaryListRowValue Value { get; set; }
+
+        /// <summary>
+        ///     Add row action. 
+        /// </summary>
+        public SummaryListRowActionViewModel Action { get; set; }
```

Comment:
Please could you change this to "Actions" to match the Nunjucks?

Reply:
Done, thanks